File: backend/stock-serv/src/main/java/org/willingfish/stock/serv/Application.java
package org.willingfish.stock.serv;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;


@SpringBootApplication
@EnableWebMvc
public class Application {
    public static void main(String[] args) {
        final SpringApplication springApplication = new SpringApplication(Application.class);
        springApplication.run(args);
    }
}

File: backend/stock-serv/src/main/kotlin/org/willingfish/stock/serv/Application.kt
package org.willingfish.stock.serv

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories
import org.springframework.web.reactive.config.EnableWebFlux

@SpringBootApplication
@EnableR2dbcRepositories
@EnableWebFlux
class Application

    fun main(args: Array<String>) {
        runApplication<Application>(*args)
    }
